src/networking/index.js

```diff
@@ -32,32 +30,32 @@ export default class {
   init(config: Config) {
     this._config = config;
 
-    this._maxSubDomain = 20;
+    this._maxSubDomain = 10;
     this._currentSubDomain = Math.floor(Math.random() * this._maxSubDomain);
-    this._providedFQDN = (this._config.secure ? 'https://' : 'http://') + this._config.origin;
     this._coreParams = {};
 
     // create initial origins
     this.shiftStandardOrigin();
   }
 
   nextOrigin(): string {
-    // if a custom origin is supplied, use do not bother with shuffling subdomains
-    if (!this._providedFQDN.match(/ps\.pndsn\.com$/i)) {
-      return this._providedFQDN;
-    }
+    const protocol = this._config.secure ? 'https://' : 'http://';
 
-    let newSubDomain: string;
+    if (!this._config.isDomainShardingEnabled()) {
+      return `${protocol}${this._config.origin}`;
+    }
 
     this._currentSubDomain += 1;
 
     if (this._currentSubDomain >= this._maxSubDomain) {
-      this._currentSubDomain = 1;
+      this._currentSubDomain = 0;
     }
```

Comment:
I think this mean internal ps.pndsn.com gets shards 1-10 and customers get 0-10? Did I read this wrong, did we want customers to get 0-9?

Reply:
The intention is to have customers use 0-9, I am not sure about our default domain tho. I'll ask on the channel